Library/Homebrew/cli/parser.rb

```diff
@@ -167,7 +167,7 @@ def initialize(cmd = nil, &block)
           @command_name = T.let(T.must(cmd_location.label).chomp("_args").tr("_", "-"), String)
           @is_dev_cmd = T.let(T.must(cmd_location.absolute_path).start_with?(Commands::HOMEBREW_DEV_CMD_PATH),
                               T::Boolean)
-          odeprecated(
+          odisabled(
             "`brew #{@command_name}'. This command needs to be refactored, as it is written in a style that",
```

Comment:
This disable message should read, "…as it is *not* written in a style that…", right?

Reply:
This was mostly a false alarm, i missed the comma and joined the strings incorrectly. If I revert `irb` to an earlier state, I get:

```
Error: Calling `brew Homebrew.irb'. This command needs to be refactored, as it is written in a style that is disabled! Use inherits from `Homebrew::AbstractCommand' ( see https://docs.brew.sh/External-Commands ) instead.
```

The last sentence is a bit unnatural, so I'll open a PR for improved readability.